apps/test/netsim/NetSimTable.js

```diff
@@ -300,247 +309,248 @@ describe("NetSimTable", function () {
         });
       });
 
-      it ("rejects negative numbers", function () {
-        assertThrows(TypeError, function () {
+      it("rejects negative numbers", function () {
+        assert.throws(function () {
           var _ = new NetSimTable(fakeChannel, 'shardID', 'tableName', {
             minimumDelayBetweenRefreshes: -88
           });
-        });
+        }, TypeError);
       });
 
-      it ("rejects `null`", function () {
-        assertThrows(TypeError, function () {
+      it("rejects `null`", function () {
+        assert.throws(function () {
           var _ = new NetSimTable(fakeChannel, 'shardID', 'tableName', {
             minimumDelayBetweenRefreshes: null
           });
-        });
+        }, TypeError);
       });
 
-      it ("rejects strings", function () {
-        assertThrows(TypeError, function () {
+      it("rejects strings", function () {
+        assert.throws(function () {
           var _ = new NetSimTable(fakeChannel, 'shardID', 'tableName', {
             minimumDelayBetweenRefreshes: "twenty hours"
           });
-        });
+        }, TypeError);
       });
 
-      it ("rejects objects", function () {
-        assertThrows(TypeError, function () {
+      it("rejects objects", function () {
+        assert.throws(function () {
           var _ = new NetSimTable(fakeChannel, 'shardID', 'tableName', {
             minimumDelayBetweenRefreshes: {}
           });
-        });
+        }, TypeError);
       });
 
-      it ("rejects Infinities", function () {
-        assertThrows(TypeError, function () {
+      it("rejects Infinities", function () {
+        assert.throws(function () {
           var _ = new NetSimTable(fakeChannel, 'shardID', 'tableName', {
             minimumDelayBetweenRefreshes: Infinity
           });
-        });
+        }, TypeError);
 
-        assertThrows(TypeError, function () {
+        assert.throws(function () {
           var _ = new NetSimTable(fakeChannel, 'shardID', 'tableName', {
             minimumDelayBetweenRefreshes: -Infinity
           });
-        });
+        }, TypeError);
       });
 
-      it ("rejects NaN", function () {
-        assertThrows(TypeError, function () {
+      it("rejects NaN", function () {
+        assert.throws(function () {
           var _ = new NetSimTable(fakeChannel, 'shardID', 'tableName', {
             minimumDelayBetweenRefreshes: NaN
           });
-        });
+        }, TypeError);
       });
     });
+    /* jshint unused: true */
   });
 
-  it ("calls readAll on the API table", function () {
+  it("calls readAll on the API table", function () {
     netsimTable.refresh(callback);
-    assertEqual(apiTable.log(), 'readAll');
+    assert.equal(apiTable.log(), 'readAll');
```

Comment:
Most of the rest of the `assert.equal` calls follow the `assert.equal(expected, actual)` pattern. This one reverses that.

However, according to the [chai documentation](http://chaijs.com/api/assert/) this is the preferred order, so I'm not sure what we want to do about that.